pymemcache/client/base.py

```diff
@@ -307,6 +307,8 @@ def set(self, key, value, expire=0, noreply=None):
                   from the cache, or zero for no expiry (the default).
           noreply: optional bool, True to not wait for the reply (defaults to
                    self.default_noreply).
+          flags: optional int, arbitrary bit field used for server-specific
+                flags
```

Comment:
Should we write something here about the new behavior when a serializer is present?

Reply:
I think put the comments in the _store_cmd() would be better since we might add flags for other functions as well.